docs/network-flow-visibility.md

```diff
@@ -286,10 +286,25 @@ will lead to a complete data loss. In that case, please refer to the
 to recover the data.
 
 ClickHouse cluster can be deployed with default Local PV or NFS PV by setting
-`clickhouse.storage.createPersistentVolume`. To have more flexibility in the
-PV creation, you can configure a customized `StorageClass` in
+`clickhouse.storage.createPersistentVolume`. To have more flexibility in the PV
+creation, you can configure a customized `StorageClass` in
 `clickhouse.storage.persistentVolumeClaimSpec`.
 
+We also offer the option of deploying Zookeeper with Persistent Volume. You can
+deploy Zookeeper with either the default Local PV or NFS PV by adjusting the
+`clickhouse.cluster.installZookeeper.storage.createPersistentVolume` setting.
+For greater flexibility in creating Persistent Volumes, you have the option to
+configure a customized StorageClass within the
+`clickhouse.cluster.installZookeeper.storage.persistentVolumeClaimSpec`.
+
+It's important to select ZooKeeper with Persistent Volume when opting to deploy
+ClickHouse with Persistent Volumes. This choice is particularly crucial because
+if you deploy ClickHouse with Persistent Volumes (PV) and use the default
+ZooKeeper, there's a risk that if the ZooKeeper pod crashes before the ClickHouse
+pod, ClickHouse will automatically switch the table to a READONLY state. This
+sequential occurrence can lead to repeated crashes of the ClickHouse pod because
+all data stored in ZooKeeper is lost in the event of a ZooKeeper crash.
```

Comment:
I'm thinking about maybe we do not need to include that many technical details?
Like just saying 

```
It's important to provision ZooKeeper with Persistent Volume when opting to deploy
ClickHouse with Persistent Volumes, as the crashing of in-memory ZooKeeper will lead
to data lost, with which the ClickHouse will keep crashing.
```

Please feel free to rephrasing it.

Reply:
Updated.